src/js/utils/html-utils.js

```diff
@@ -109,15 +142,60 @@
                         },
                         data
                     );
-                    return rawTemplate(augmentedData);
+                    return HTML(rawTemplate(augmentedData));
                 };
             return template;
         };
 
+        /**
+         * A wrapper for $.html that safely escapes the provided HTML.
+         *
+         * If the HTML is provided as an HTML snippet then it is set directly
```

Comment:
I think you should include this explanation in the other two methods below as well.